packages/mdc-select/index.ts

```diff
@@ -502,10 +502,14 @@ class MDCSelect extends MDCComponent<MDCSelectFoundation> implements RippleCapab
    */
   private getNormalizedXCoordinate_(evt: MouseEvent | TouchEvent): number {
     const targetClientRect = (evt.target as Element).getBoundingClientRect();
-    const xCoordinate = (evt as MouseEvent).clientX; // TODO(acdvorak): How does this work for TouchEvent?
+    const xCoordinate = this.isTouchEvent_(evt) ? evt.touches[0].clientX : evt.clientX;
     return xCoordinate - targetClientRect.left;
   }
 
+  private isTouchEvent_(evt: MouseEvent | TouchEvent): evt is TouchEvent {
```

Comment:
what if we changed this to just `getXCoordinateFromEvent`? Then we just encapsulate this `this.isTouchEvent_(evt) ? evt.touches[0].clientX : evt.clientX;` logic...cause we still end up having to cast it anyways inside this method.

Reply:
As discussed offline, it would need to look like this:

```ts
  private getNormalizedXCoordinate_(evt: MouseEvent | TouchEvent): number {
    const targetClientRect = (evt.target as Element).getBoundingClientRect();
    return this.getClientXFromEvent_(evt) - targetClientRect.left;
  }

  private getClientXFromEvent_(evt: MouseEvent | TouchEvent): number {
    const touchEvt = evt as TouchEvent;
    const mouseEvt = evt as MouseEvent;
    if (touchEvt.touches) {
      return touchEvt.touches[0].clientX;
    }
    return mouseEvt.clientX;
  }
```

Matt and I prefer the `isTouchEvent_()` style, so we're going to leave it as-is.